### ISSUE TITLE
Floated images not displaying side by side

### ISSUE THREAD

**Issue by mnaydan** [Princeton-CDH/ppa-django#326]
Bug noticed after 3.3.0 update.

Consecutive floated images are displaying staggered, leading to other formatting issues, including some captions and images displaying out of order. See Gissoo's essay: https://prosody.princeton.edu/editorial/2019/05/designing-ppa-user-interface-interactions/

**Comment by mnaydan** [comment]
This is working! Images are now floating side by side. I tested with floated images surrounded by text blocks, 2 consecutive rows of floated images, a floated gif / image side by side, and floated images followed by a full width image.

1 hitch I found - when floated images are followed by a full width image, it appears correctly. But when floated images are followed by a full width *gif*, the gif doesn't appear full-width. See screenshot. Can this be fixed before we close out? 

<img width="466" alt="Screen Shot 2020-04-30 at 2 23 48 PM" src="https://user-images.githubusercontent.com/22945600/80746209-7e5c0c80-8aef-11ea-813d-4cb3eb5ea98f.png">
